versions/v1/units.json

```
@@ -10457,5 +10457,42 @@
     },
     "source": null,
     "sourceReference": null
+  },
+  {
+    "externalId": "attenuation:decib",
+    "name": "DeciB",
+    "quantity": "Attenuation",
+    "longName": "Decibel",
+    "aliasNames": [
+      "DB",
+      "-DB",
+      "-dB",
+      "dB",
+      "Decibel"
+    ],
+    "symbol": "dB",
+    "conversion": {
+        "multiplier": 1.0,
+        "offset": 0.0
+    },
+    "source": "qudt.org",
+    "sourceReference": "https://qudt.org/vocab/unit/DeciB" 
+  },
+  {
+    "externalId": "attenuation:b",
+    "name": "Bel",
```

```suggestion
    "name": "B",
```
Here we'll have to change the name to `B`, to match the name in qudt. The `longName` can continue as is.